cmd/machine-api-operator/start.go

```diff
@@ -48,22 +49,25 @@ func init() {
 	rootCmd.AddCommand(startCmd)
 	startCmd.PersistentFlags().StringVar(&startOpts.kubeconfig, "kubeconfig", "", "Kubeconfig file to access a remote cluster (testing only)")
 	startCmd.PersistentFlags().StringVar(&startOpts.imagesFile, "images-json", "", "images.json file for MAO.")
+
+	klog.InitFlags(nil)
+	flag.Parse()
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
```

Comment:
Why are we adding pflag here? Should we not just be using the flag package?

Reply:
No, apparently cobra overrides the flags and needs to pass klog defaults this way. This was the cause for the CI failures